samples/kotlin/face/FaceAnalyzerSample/app/build.gradle

```diff
@@ -52,16 +52,18 @@ android {
 }
 
 dependencies {
+    implementation group: 'net.sourceforge.streamsupport', name: 'android-retrofuture', version: '1.7.4'
     implementation 'com.google.android.material:material:1.7.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
     implementation 'com.azure.android:azure-core-http-httpurlconnection:1.0.0-beta.10'
     implementation 'com.azure.android:azure-core-credential:1.0.0-beta.10'
     implementation 'com.azure:azure-ai-vision-common-internal:0.15.1-beta.1'
-    implementation 'androidx.appcompat:appcompat:1.6.1'
     testImplementation 'junit:junit:4.+'
     androidTestImplementation 'androidx.test.ext:junit:1.1.3'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
     implementation "androidx.activity:activity:1.6.1"
-    implementation "com.azure.ai:azure-ai-vision-common:0.16.0-beta.1"
-    implementation "com.azure.ai:azure-ai-vision-faceanalyzer:0.16.0-beta.1"
+    def camerax_version = "1.1.0"
+    implementation "androidx.camera:camera-camera2:$camerax_version"
+    implementation "androidx.camera:camera-lifecycle:$camerax_version"
+    implementation fileTree(dir: '../libs', include: ['*.aar'])
```

Comment:
this should be updated to use com.azure.ai:azure-ai-vision-faceanalyzer:0.16.1-beta.1

Reply:
fixed.